_includes/specials/taproot/00-bech32m.md

```diff
@@ -0,0 +1,42 @@
+## Preparing for taproot #1: bech32 sending support
+{% assign trb = "709,632" %}{% comment %}<!-- stop me from mistyping the number -->{% endcomment %}
+
+*The first segment in a weekly series about how developers and service providers
+can prepare for the upcoming activation of taproot at block height
+{{trb}}.*
+
+Starting at block {{trb}}, expected in November, Bitcoin users will be
+able to safely receive payments to taproot  addresses.  Given the
+user enthusiasm for taproot and the five months that wallet
+developers have to implement support for it, Optech expects there to be
+several popular wallets that will allow their users to generate taproot
+addresses at the earliest possible moment.
+
+That means any other wallet or service that sends bitcoins to
+user-provided addresses needs to be able to Pay To TapRoot (P2TR) by
```

Comment:
Using Pay To Taproot/P2TR as a verb/phrase is a little confusing, and I think we should discourage it in general. We don't say "the sender paid to script hash" but rather "the sender/wallet created a P2SH output" or "the receiver generated a P2SH address".

Here, I'd suggest changing the sentence to read "needs to be able to send to P2TR (Pay To TapRoot) addresses" or similar.